scout/models/variant/variant.py

```diff
@@ -98,7 +98,7 @@
 clinsig = dict(value=int, accession=str, revstat=str)
 
 gt_call = dict(
-    sample_id=str,
+    individual_id=str,
```

Comment:
Hm, indeed: you are not wrong. But, if you want to change it throughout you would need to track it down through the `build` path as well. And if you actually change the name in `build`, keep going, because some other stuff like `is_affected` would break. Up to you, but I would not mind e.g. just adding `individual_id` to the model and making an issue for it rather than adding something that may have consequences that are hard to test?

Reply:
Took a closer look, and realized I was wrong about this. Got confused because `gt_call` is being used as a name for both pre and post build step objects.

Reply:
I would not say wrong, but those are very annoying. If we ever have a lot of developer time, refactoring that mechanism and making nice clear storage objects and a couple of kinds of lightly and heavily populated display objects is definitely on the wish list.